web/src/components/UserAvatar.tsx

```diff
@@ -45,15 +46,29 @@ export const UserAvatarDetails: React.FC<UserAvatarProps> = ({
   tooltip,
   ...rest
 }) => {
+  const Component = tooltip ? TooltipAvatar : Avatar;
+
+  //skip lookup user step if creating user was terraform
+  if (user === "bot_governance_api") {
+    // Loading/loaded states
+    return (
+      <HStack>
+        <Avatar
+          icon={<TerraformIcon fontSize="0.9rem" />}
+          {...rest}
+          bg="white"
+        />
+        <Text {...textProps}>Terraform</Text>
+      </HStack>
+    );
+  }
   // @ts-ignore
   const { data, isValidating } = useGetUser(user);
```

Comment:
Could you please refactor this component so that the `useGetUser` hook is not conditionally called? (sorry, I'm pretty sure I initially wrote this component too!). @jordiup can give you a hand if you need.

The problem with this component is that if the `user` prop is equal to "bot_governance_api", the component returns early and the `useGetUser` hook is never called. This breaks one of the [Rules of Hooks](https://reactjs.org/docs/hooks-rules.html): "always use Hooks at the top level of your React function, before any early returns".

We'll need to split this component up into a `UserAvatar` and a `BotAvatar` component I think, and have a top level `CFAvatar` component that checks the `name` prop and renders one of the two